modules/ting_subsearch/ting_subsearch_secondary/templates/ting-subsearch-secondary--message-box.tpl.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Secondary search item.
+ *
+ * @var $items
+ * @var $message
+ * @var $secondary_link
+ * @var $secondary_link_text
+ */
+?>
+<div id="ting-subsearch-secondary" class="message-box search-field-in-content--message-tss">
+  <div class="message-box--title">
+    <?php print $message; ?>
+  </div>
+  <div class="message-box--list-items">
+  <?php print $items; ?>
+  </div>
+  <div class="message-box--external-search">
+    <a href="<?php print $secondary_link; ?>" target="_blank">
+      <?php print $secondary_link_text; ?>
+    </a>
+  </div>
+</div>
```

Comment:
I am not too fond of the classes and ids used here.

1. `id="ting-subsearch-secondary"` never seems to be referenced anywhere. Can it be dropped?
2. I prefer namespacing BEM blocks with their responsible modules. `message-box` is quite generic. `ting-subseearch-messages` avoids introducing generic names in specific contexts. It also provides quick information about which module is responsible for the element.
3. `search-field-in-content--message-tss` seems like a block element used without a block. In general I found it hard to find references to `search-field-in-content` in the codebase. Could we do without it?

Reply:
Thanks for the input 👍 Will update ids and classes

Reply:
Remove unused classses and ids. Updated BEM-structure.